tests/CppWinRTTests.cpp

```diff
@@ -182,6 +184,44 @@ TEST_CASE("CppWinRTTests::VectorToVector", "[cppwinrt]")
     winrt::uninit_apartment();
 }
 
+TEST_CASE("CppWinRTTests::BufferToArrayView", "[cppwinrt]")
+{
+    std::array<int32_t, 2> testData = { 314159265, 27182818 };

```

Comment:
```suggestion
    winrt::init_apartment();
    auto uninit = wil::scope_exit([]{ winrt::uninit_apartment(); });

    std::array<int32_t, 2> testData = { 314159265, 27182818 };
```

This seems to avoid the crash. Note that I still hit the crash locally for other tests, however AFAIK we haven't hit the same in the build labs

Reply:
OMG! This comment exists for the CppWinRTAuthoringTests and I didn't even know about it!:
```c++
// This test cannot run in the same process as the malloc spies tests in wiTest.cpp
// MSFT_internal: https://task.ms/44191550
```
Making that `[LocalOnly]` isn't really the correct move, though...

Reply:
@dunhor Can we proceed with this PR? We sort of gave up on the NotifyPropertyChanged test (PRs 368, 374).

Reply:
I've disabled the NotifyPropertyChanged test to run during CI. We can either do the same thing here or we can update the test to call `CoInitialize` to avoid the crash during shutdown.

Reply:
Err, actually disabling for just CI is probably not the best move. If we go the disable route, we should disable even when run locally so that the scripts don't break.